src/v3/src/transformer/layout/idxTransformerMapping.ts

```diff
@@ -242,23 +247,27 @@ const TransformerMap: {
       transform: transformEmailAuthenticatorEnroll,
       buttonConfig: { showDefaultSubmit: false },
     },
+    [AUTHENTICATOR_KEY.GOOGLE_OTP]: {
+      transform: transformGoogleAuthenticatorEnroll,
+      buttonConfig: { showDefaultSubmit: false },
+    },
     [AUTHENTICATOR_KEY.PASSWORD]: {
       transform: transformEnrollPasswordAuthenticator,
     },
     [AUTHENTICATOR_KEY.PHONE]: {
       transform: transformPhoneCodeEnrollment,
       buttonConfig: { showDefaultSubmit: false },
     },
-    [AUTHENTICATOR_KEY.WEBAUTHN]: {
-      transform: transformWebAuthNAuthenticator,
+    [AUTHENTICATOR_KEY.RSA]: {
+      transform: transformRSAAuthenticator,
       buttonConfig: { showDefaultSubmit: false },
     },
```

Comment:
No direct bug here but it looks like RSA is added twice so you can remove one of them

Reply:
Ah, its the way github is displaying it but one is for Enroll and the other is for Challenge. It also didn't help that I also re-ordered some of the others (to make it alphabetical) 😅